packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -231,10 +231,10 @@ location ~ ^/service/(?<service_path>.+) {
     set $original_uri $uri;
 
     # Give services a lot of leeway. A particular extreme is for uploading
-    # packages to a package registry. The Jupyter package is one of our
-    # largest at just under 10GB. Use 16GB as the limit to allow for this to
+    # packages to a package registry. The data-science-engine package is one
+    # of our largest at over 16GB. Use 32GB as the limit to allow for this to
     # expand.
-    client_max_body_size 16384m;
+    client_max_body_size 32768m;
```

Comment:
I'm tempted to keep this as 24GB. Although it is slightly painful for customers when we hit the limit, it is useful to discover when package sizes actually reach these extremes. And these files can be modified on a per-customer basis when necessary.